src/narada/__init__.py

```diff
@@ -16,9 +16,6 @@
     ResponseContent,
 )
 
-__version__ = "0.1.11"
```

Comment:
There is actually no reason we need this. Modern Python can use `importlib.metadata.version("narada")` to get this information.